conftest.py

```diff
@@ -1,3 +1,67 @@
+# tests directory-specific settings - this file is run automatically
```

Comment:
```suggestion
# coding=utf-8
# Copyright 2020 The HuggingFace Team. All rights reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

# tests directory-specific settings - this file is run automatically
```

why did you remove the license, please add it back

Reply:
I see what's happening. I didn't mean to remove the license per say, the current file on OH is [https://github.com/huggingface/optimum-habana/blob/main/conftest.py] , I just included the parts that were important for the `test_utils`.  Great catch!